common/static/coffee/spec/discussion/view/thread_response_show_view_spec.coffee

```diff
@@ -115,7 +115,7 @@ describe "ThreadResponseShowView", ->
         expect(@view.$(".posted-details").text()).not.toMatch("marked as answer")
 
     it "allows a moderator to mark an answer in a question thread", ->
-        DiscussionUtil.loadRoles({"Moderator": parseInt(window.user.id)})
+        DiscussionUtil.loadRoles({"Moderator": [parseInt(window.user.id)]})
```

Comment:
Note to reviewers: Subtle change in behavior with _.union (doesn't handle non-array inputs).